app/src/main/res/layout/welcome.xml

```diff
@@ -43,7 +43,7 @@
         android:layout_height="wrap_content"
         android:fontFamily="sans-serif"
         android:text="@{viewModel.computeProfileNameText()}"
-        android:textColor="@color/component_color_home_activity_greeting_text_color"
+        android:textColor="@color/component_color_shared_content_item_text_view_color"
```

Comment:
Remove this and use correct color `component_color_shared_content_item_text_view_color`

Reply:
Done.